vidarr-cli/src/main/java/ca/on/oicr/gsi/vidarr/cli/TestValidatorScript.java

```diff
@@ -101,29 +112,35 @@ public boolean validate(String id) {
   }
 
   @JsonProperty("metrics_calculate")
-  public String getMetricsCalculate() {
+  public String getMetricsCalculate()
+  {
```

Comment:
Please revert style changes and refer to https://github.com/oicr-gsi/vidarr/blob/master/code-style.md